src/application.rs

```diff
@@ -290,6 +301,16 @@ impl SwayOSDApplication {
 						};
 						(ArgTypes::DeviceName, Some(value))
 					}
+					"top-margin" => {
```

Comment:
Setting this will allow users to specify a per osd window margin (`swayosd --caps-lock --top-margin 0`), but this sets it globally. I'm not really sure how to tackle this.

We could only allow to set the top-margin when starting swayosd but then setting a per osd margin wouldn't work. Any ideas?

Reply:
Good catch!

> We could only allow to set the top-margin when starting swayosd

I think this seems most reasonable. I do not see any real value in having different OSD windows on different heights.

Reply:
Small addendum here.. I think I like how it is right now. While testing different margin options it was convinient to just run `swayosd --top-margin ...` to change it on the fly without having to restart the process.